docs/vocascan-server/installation/pm2/pm2.md

```diff
@@ -103,5 +105,6 @@
 
    ?> Info: To see every registered user use this [command]("vocascan-server/cli#list")
 
-   Your setup is now ready and you can start using your server. Please remember that the admin user you just created should not be used as a regular account. Please create another one for your daily use.
```

Comment:
You deleted that line entirely. I think this line should still stay in the docs, but not in an infobox. 
>  Your setup is now ready and you can start using your server.